src/applications/hca/components/DemographicField.jsx

```diff
@@ -14,9 +14,9 @@ const DemographicField = props => {
       <>
         <div className="review-row">
           <dt>{uiSchema['ui:title']}</dt>
-          {categories.length > 0 && (
-            <dd>{uiSchema[categories[0]]['ui:title']}</dd>
-          )}
+          <dd>
+            {categories.length > 0 && uiSchema[categories[0]]['ui:title']}
```

Comment:
Won't this display `false` if the length is zero?

Reply:
I don't think this will render anything if length is zero. The only gotcha with this type of thing is making sure that you actually compare `.length` to zero